src/lib.rs

```diff
@@ -184,16 +167,22 @@ impl TimerFd {
     /// This can also fail in various cases of resource exhaustion. Please check
     /// `timerfd_create(2)` for details.
     pub fn new_custom(clock: ClockId, nonblocking: bool, cloexec: bool) -> IoResult<TimerFd> {
-
-        let mut flags = 0;
+        let mut flags = TimerfdFlags::empty();
         if nonblocking {
-            flags |= TFD_NONBLOCK;
+            flags |= TimerfdFlags::NONBLOCK;
         }
         if cloexec {
-            flags |= TFD_CLOEXEC;
+            flags |= TimerfdFlags::CLOEXEC;
         }
 
-        let fd = neg_is_err(unsafe { timerfd_create(clock as libc::c_int, flags) })?;
+        let clock = match clock {
+            ClockId::Realtime => TimerfdClockId::Realtime,
+            ClockId::RealtimeAlarm => TimerfdClockId::RealtimeAlarm,
+            ClockId::Monotonic => TimerfdClockId::Monotonic,
+            ClockId::Boottime => TimerfdClockId::Boottime,
+            ClockId::BoottimeAlarm => TimerfdClockId::BoottimeAlarm,
+        };
```

Comment:
This redundancy is regrettable. I wonder if it's possible to just export `TimerfdClockId` as `ClockId` instead without breaking anything?

Reply:
I expect it wouldn't break anything, though it would make rustix a public dependency of rust-timerfd, which would be nice to avoid, so that rust-timerfd doesn't need a semver bump whenever rustix has a semver bump.